src/promptflow/promptflow/_cli/_pf/entry.py

```diff
@@ -71,13 +74,16 @@ def entry(argv):
         elif args.action == "tool":
             dispatch_tool_commands(args)
```

Comment:
shall we set the SUCCESS inside try?

Reply:
ok

Reply:
You can set a default failed status, so we can discover there are unexpected cases

Reply:
done